django_meili/tests.py

```diff
@@ -174,3 +174,52 @@ def test_django_meili_does_not_sync_when_offline(self):
 
         self.assertEqual(PostNoGeo.meilisearch.count(), post_no_geo_original_count)
         self.assertEqual(Post.meilisearch.count(), post_updated_count)
+
+
+
+@override_settings(MEILISEARCH={"SYNC": True}, DEBUG=True)
+class DjangoMeiliNonStandardIdTestCase(TestCase):
+    target_model = NonStandardIdPost
```

Comment:
I like this target_model idea, but it shouldn't be limited to just here. Don't worry about changing it now, but I think I'll carry this pattern to the rest of the test cases.

Yeah, I can create a base test case for them, clear up some redundancies.